ajax/libs/fast-xml-parser/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "fast-xml-parser",
+  "version": "2.2.1",
```

Comment:
@clairetsai818 Yes I think so if the auto-update config still works in latest version.